pkg/controllers/syncer/context/context.go

```diff
@@ -27,6 +27,10 @@ type RegisterContext struct {
 	Options     *controllercontext.VirtualClusterOptions
 	Controllers sets.String
 
+	// TargetNamespace is the namespace in the host cluster where
+	// vcluster should sync namespaced scope objects to
+	TargetNamespace string
```

Comment:
Please get rid of this as this couldn't be used correctly with multi-namespace syncer as we don't want to replace the labels there